src/app/shared/pipes/capitalise-pipe.ts

```diff
@@ -16,9 +16,24 @@ export class Capitalise implements PipeTransform {
   /**
    * This method will get a string and return it with its first letter capitalised.
    * @param str The string to capitalise
+   * @param lowercaseStartingString Whether we should convert the string to lowercase first. Default is false.
+   * @param capitaliseEveryWord Whether we should capitalise every word in the string. Default is false.
    * @returns The capitalised string
    */
-  transform(str: string): string {
+  transform(
+    str: string,
+    lowercaseStartingString: boolean = false,
+    capitaliseEveryWord: boolean = false,
+  ): string {
+    if (lowercaseStartingString) {
+      str = str.toLowerCase();
+    }
+    if (capitaliseEveryWord) {
+      return str
+        .split(' ')
+        .map((word) => word.substring(0, 1).toUpperCase() + word.substring(1))
+        .join(' ');
+    }
     return str.substring(0, 1).toUpperCase() + str.substring(1);
```

Comment:
In our metadata, I think the terms are all caps with underscores instead of spaces

So we must also replace underscores. The terms in DUO are all lowercase, I think we should output them like that, not try to capitalize them.